pkg/flexvolume/flexvolume.go

```diff
@@ -67,6 +67,8 @@ const (
 	OptionKeyPodUID       = "kubernetes.io/pod.uid"
 
 	OptionKeyServiceAccountName = "kubernetes.io/serviceAccount.name"
+
+	DefaultSymlinkDirectory = "oci-bvs"
```

Comment:
`pkg/flexvolume` is intended to be vendor agnostic and should not contain OCI specific code. Suggest moving to `cmd/oci`. Additionally, I'd suggest `DefaultDriver` would be a more appropriate name for this constant.